examples/query/react/basic/package.json

```diff
@@ -13,7 +13,8 @@
     "react-scripts": "5.0.1"
   },
   "devDependencies": {
-    "@testing-library/react": "^13.3.0",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/react": "^16.0.1",
```

Comment:
Honestly, I didn’t get how to run tests in examples, and how to check if everything is OK.

When I run `yarn test` in the master branch, the tests run only for the following packages:

- `@reduxjs/rtk-codemods`
- `@rtk-query/codegen-openapi`
- `@rtk-query/graphql-request-base-query`
- `@reduxjs/toolkit`

And when I go to `examples/query/react/basic` and run `npm test` there, I get the following error:

```
> @examples-query-react/basic@1.0.0 test
> react-scripts test --runInBand

TypeError: Cannot read properties of undefined (reading 'EXTENSION')
    at Runtime.createHasteMap (rtk/node_modules/jest-runtime/build/index.js:547:43)
    at rtk/node_modules/@jest/core/build/cli/index.js:231:55
    at Array.map (<anonymous>)
    at buildContextsAndHasteMaps (rtk/node_modules/@jest/core/build/cli/index.js:228:13)
    at _run10000 (rtk/node_modules/@jest/core/build/cli/index.js:305:47)
    at runCLI (rtk/node_modules/@jest/core/build/cli/index.js:173:9)
    at async Object.run (rtk/node_modules/jest-cli/build/cli/index.js:155:37)
npm error Lifecycle script `test` failed with error:
npm error code 1
npm error path rtk/examples/query/react/basic
npm error workspace @examples-query-react/basic@1.0.0
npm error location rtk/examples/query/react/basic
npm error command failed
npm error command sh -c react-scripts test --runInBand
```

Reply:
That should be fixed by #4603.

Reply:
Should I correct something in this regard in the current pull request?